src/Common/TurbulenceClosures/TurbulenceClosures.jl

```diff
@@ -402,6 +405,81 @@ function turbulence_tensors(
     return ν, D_t, τ
 end
 
+# ### [Viscous Sponge](@id viscous-sponge)
+# `ViscousSponge` requires a user to specify a constant viscosity (kinematic), 
```

Comment:
w.r.t line 427, dynamic this should be "dynamic visc..." (?)

Reply:
correct